src/Rector/AbstractDrupalCoreRector.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DrupalRector\Rector;
+
+use Drupal\Component\Utility\DeprecationHelper;
+use DrupalRector\Contract\DrupalCoreRectorInterface;
+use PhpParser\Node;
+use PhpParser\Node\Expr\ArrowFunction;
+use Rector\Core\Rector\AbstractRector;
+
+abstract class AbstractDrupalCoreRector extends AbstractRector implements DrupalCoreRectorInterface
+{
+    public function refactor(Node $node)
+    {
+        $drupalVersion = str_replace(['.x-dev', '-dev'], '.0', \Drupal::VERSION);
+        if (version_compare($drupalVersion, $this->getVersion(), '<')) {
```

Comment:
My brain says: shouldn't `.0` be `.9999` but I don't know. I guess its best to assume `.0` because we don't know where or what the `dev` is at. It could be an old HEAD. For GitLab CI/DrupalCI `drupal/core` isn't always a tagged release, will this break project update bot because it'll skip a lot of things?

Reply:
Core policy defines that depreactions can only be added in minors, see https://www.drupal.org/about/core/policies/core-change-policies/allowed-changes#minor

So lets see:
Depraction introduced in 10.1.0 should evalutate to false on anything on or after that version. This means that lower than 10.1.0 is correct. Since if you are running on 10.1.x you will evaluate to false.